README.md

```diff
@@ -144,6 +144,20 @@ TF_CLI_CONFIG_FILE=./local_dev.tfrc terraform plan
 
 Alternatively you can add the provider configuration to the `$HOME/.terraformrc` file
 
+### Generate automatic documentation
+
+Documentation of the provider is based on official Terraform documentation
+plugin: https://github.com/hashicorp/terraform-plugin-docs
+
+```
+go install github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs@latest
+tfplugindocs
+```
+
+By running the process, it will scan all resources metadata plus the examples
+directory to generate a complete documentation structure in the `docs/`
+directory.
```

Comment:
Is it mandatory to execute this command before releasing a new version? If yes, I would add a step in the "Release a New Version" section

Reply:
Done

Reply:
Mmmmh is it done? 🤔 
I mean, I don't see any mention of the doc generation in the "Release a New Version" section of the README. Does it mean that we don't need to execute this when releasing a new version of this provider?

Reply:
Oh I put it in the Notion page

Reply:
Oh OK. It feels super weird as for all other projects instructions are in the README. Is there a reason for this? I'm not keen on having the instructions in Notion, especially because there is no sensitive information in these instructions. Anyway I'm approving the PR